src/pages/Home/index.js

```diff
@@ -29,7 +33,7 @@ export default function Home(){
             })
     }
     return (
-        <ColumnsLayout>
+        <ColumnsLayout title={`${t('page-title')} - Instant Visio`}>
             <Description />
             <>
```

Comment:
not needed

Reply:
If I remove it, and I go from Blog page back to Home page, document title is still "Blog - Instant Visio"